docs/release-notes/Percona-Everest-1.1.0-(2024-08-12).md

```diff
@@ -12,6 +12,14 @@ Version 1.1.0 introduces the following changes:
 !!! info "Important"
     Percona Everest 1.1.0 comes with its own set of [limitations](Percona-Everest-1.1.0-(2024-08-12).md#known-limitations) that you should be aware of.
 
+### Breaking change in Percona Everest 1.1.0
```

Comment:
I wouldn't call this a breaking change
I'd say use a info message or something saying upgrade instructions or something

Reply:
ok